test/lib/samson/time_sum_test.rb

```diff
@@ -15,7 +15,7 @@
     it "records single" do
       result = Samson::TimeSum.record("sql.active_record" => :db) { User.first }
       result.keys.must_equal [:db]
-      assert result[:db].between?(0, 10), result
+      assert result[:db].between?(0, 100), result
```

Comment:
what does this test actually do?

Reply:
make sure the time counting returns a sane value (ms vs s) so <1000 is fine 🤷‍♂